### ISSUE TITLE
Request for adding Odido

### ISSUE THREAD

**Issue by xxsuperdreamxx** [simple-icons/simple-icons#12442]
### Brand Name

Odido

### Website

https://www.odido.nl/

### Popularity Metric

The similar web rank is 7,865 https://www.similarweb.com/website/odido.nl

Odido is the new T-Mobile Rebrand for the Netherlands

### Official Resources for Icon and Color

Here are the official Brand colors
https://merk.odido.nl/en/colour

The Logo is divided into 3 types https://merk.odido.nl/en/logo

The Color logo Using (Glow One, Glow Two and Glow Three and Glow Four from the color palette)
![odido-logo 1](https://github.com/user-attachments/assets/57a6ae77-6207-4a6a-80fd-68a380f8ecbb)
Original Source https://www.odido.nl/Assets/Lfwd/Static/odido-logo.svg

The Solid Black logo
![odido-logo-black 1](https://github.com/user-attachments/assets/418ea5f7-ed8e-4f46-855f-6dcd80321d88)

Original Source https://www.odido.nl/Assets/Lfwd/Static/odido-logo-black.svg

The Solid White logo
![odido-logo-white 1](https://github.com/user-attachments/assets/f28bce13-d4a3-427d-8721-ee42334cf6f9)

Original Source https://www.odido.nl/Assets/Lfwd/Static/odido-logo-white.svg

### Additional Comments

The Color Logo is the official logo used everywhere but the other two alternatives are accepted by Odidio in general

**Comment by PeterShaggyNoble** [comment]
Thanks for the request, @xxsuperdreamxx. Would you be able to check for us if there is anything in any terms & conditions, or equivalent, that might require us to seek permission to add this, or prevent us from adding it at all?

**Comment by xxsuperdreamxx** [comment]
> Thanks for the request, @xxsuperdreamxx. Would you be able to check for us if there is anything in any terms & conditions, or equivalent, that might require us to seek permission to add this, or prevent us from adding it at all?

No, there is no issue of using it. As long as peoole follow their guidelines. Aka not to use it for impersonation. And they are not allowed to modify the logo.

Also odido added few more logo types like outlined for example
https://dam.odido.nl/web/52967ad9a20e29eb/b2c-logos---downloads-brand-portal/?viewType=grid

I am leaving them here if you also want to check them out.


Thank you again for takimg the time

**Comment by PeterShaggyNoble** [comment]
Thanks, @xxsuperdreamxx. Would you like to have a read through [our contributing guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon) and try this one yourself?

**Comment by xxsuperdreamxx** [comment]
Hey @PeterShaggyNoble done and done. I made sure to check everything with the lint it gave me big thumbs up so It should be fine
